libvast/src/detail/mmapbuf.cpp

```diff
@@ -95,7 +95,7 @@ size_t mmapbuf::size() const {
 bool mmapbuf::resize(size_t new_size) {
   // Also fail if we created a private mapping of a file not opened RW.
   // For now, users cannot control these aspects.
-  if (!map_ || offset_ > new_size)
+  if (!map_)
```

Comment:
This appears to be an orthogonal fix. Technically, it should go into a separate commit.